### DISCUSSION TITLE
Version 1.23.0

### DISCUSSION THREAD

**Discussion by apple502j** [ScratchAddons/ScratchAddons#3879] [Changelog]
## Bugfixes
- [#3710](https://github.com/ScratchAddons/ScratchAddons/issues/3710) dark-www does not make scrollbars in forum previews dark. (prio-4)
- [#3710](https://github.com/ScratchAddons/ScratchAddons/issues/3710) dark-www adds extra margin between projects in My Stuff. (prio-4)
- [#3710](https://github.com/ScratchAddons/ScratchAddons/issues/3710) dark-www causes eye icon on My Stuff to look weird. (prio-4)
- [#3710](https://github.com/ScratchAddons/ScratchAddons/issues/3710) dark-www makes Become a Scratcher screen unreadable. (prio-4)
- [#3823](https://github.com/ScratchAddons/ScratchAddons/issues/3823) account-settings-capitalize does not work inside the editor. (prio-4)
- [#3838](https://github.com/ScratchAddons/ScratchAddons/issues/3838) progress-bar allows bar size that is too high or too low. (prio-4)
- [#3839](https://github.com/ScratchAddons/ScratchAddons/issues/3839) editor-dark-mode has an outdated preset for TurboWarp Dark. (prio-3)
- [#3840](https://github.com/ScratchAddons/ScratchAddons/issues/3840) my-ocular reaction menu buttons are outlined on Chrome. (prio-4)
- [#3844](https://github.com/ScratchAddons/ScratchAddons/issues/3844) emoji-picker emojis are outlined on Chrome. (prio-3)
- [#3851](https://github.com/ScratchAddons/ScratchAddons/issues/3851) block-cherry-picking causes comments attached to other blocks to be dragged. (prio-3)
- [#3851](https://github.com/ScratchAddons/ScratchAddons/issues/3851) block-cherry-picking breaks when dragging and dropping the first block of a script. (prio-3)
- [#3852](https://github.com/ScratchAddons/ScratchAddons/issues/3852) scratchr2 causes borders on the selected featured project to disappear. (prio-4)
- [#3866](https://github.com/ScratchAddons/ScratchAddons/issues/3866) customize-avatar-border shows the outline color setting when inapplicable. (prio-4)
- [#3876](https://github.com/ScratchAddons/ScratchAddons/issues/3876) onion-skinning breaks when zoomed in. (prio-3)
- [#3876](https://github.com/ScratchAddons/ScratchAddons/issues/3876) onion-skinning is laggy. (prio-3)
- [#3881](https://github.com/ScratchAddons/ScratchAddons/issues/3881) project-info shows incorrect block count after loading a project. (prio-4)
- [#3888](https://github.com/ScratchAddons/ScratchAddons/issues/3888) dark-www does not change the Hour of Code banner. (prio-4)
- [#3892](https://github.com/ScratchAddons/ScratchAddons/issues/3892) editor-comment-previews triggers when hovering over the line between blocks and comments. (prio-3)
- [#3892](https://github.com/ScratchAddons/ScratchAddons/issues/3892) editor-comment-previews errors when resizing comments. (prio-4)
- [#3908](https://github.com/ScratchAddons/ScratchAddons/issues/3908) better-img-uploads does not scale images properly. (prio-4)

## New Features
- [#3710](https://github.com/ScratchAddons/ScratchAddons/issues/3710) dark-www: add customizable settings and presets (prio-3)
- [#3710](https://github.com/ScratchAddons/ScratchAddons/issues/3710) editor-dark-mode: update Experimental Dark preset (prio-3)
- [#3731](https://github.com/ScratchAddons/ScratchAddons/issues/3731) New addon: script-snap (prio-3)
- [#3769](https://github.com/ScratchAddons/ScratchAddons/issues/3769) New addon: hi-res-thumbnails (prio-3)
- [#3795](https://github.com/ScratchAddons/ScratchAddons/issues/3795) Split project-info addon and add block-count addon. (prio-3)
- [#3811](https://github.com/ScratchAddons/ScratchAddons/issues/3811) UI for new features and updates (prio-3)
- [#3804](https://github.com/ScratchAddons/ScratchAddons/issues/3804) fix-editor-comments: add comment snapping and leashing (prio-3)
- [#3825](https://github.com/ScratchAddons/ScratchAddons/issues/3825) forum-id: automatically add link (prio-3)

**Comment by Mbrick2** [comment]
Release Date?

**Reply by Dhuls-Scratch** [reply]
Originally WorldLanguages said Dec 7th, but they later said

> We might need some more time because of all the migrations tho :P

Code for one migration exists, the other's method still hasn't been decided (I think)
